### PR TITLE
Remove old code and update Readme

### PR DESCRIPTION
**Pull request by jesica-fera**

### CODE REVIEW

**Comment by uip-robot-zz** [comment]
Git2Gus App is installed but the `.git2gus/config.json` doesn't exist.

File: README.md [hunk 1]
```diff
@@ -1,28 +1,15 @@
 # DataWeave LSP
 
-This project contains the first basic implementation of a DataWeave LSP
+The code for this project has been moved to a private repository. This repository will be left open to the community to
```
Comment (afelisatti) [inline README.md]:
```suggestion
This repository is open to the community to 
```

File: README.md [hunk 2]
```diff
@@ -1,28 +1,15 @@
 # DataWeave LSP
 
-This project contains the first basic implementation of a DataWeave LSP
+The code for this project has been moved to a private repository. This repository will be left open to the community to 
+help us improve it and report issues.
```
Comment (afelisatti) [inline README.md]:
```suggestion
help us improve our Visual Studio Code support and report issues.
```

File: README.md [hunk 3]
```diff
@@ -1,28 +1,15 @@
...
-
-3. Release a version
-
-In order to release just create a tag with v<next_version> example if we want to release 0.0.4
-we just need to create v0.0.4 and push the tag. The CI should take care of the rest
-
-
-4. How to copy 
\ No newline at end of file
+Your feedback is appreciated, if you have any issues, enhancements, or feature requests to report, please feel free to create Github Issues in this repository and they will be timely addressed.
```
Comment (afelisatti) [inline README.md]:
```suggestion
Your feedback is appreciated, if you have any issues, enhancements, or feature requests to report, please feel free to create Github Issues in this repository and they will be timely reviewed. 
```

File: README.md (hunk 1 above)
Comment (afelisatti) [reply README.md]: I wouldn't share this broadly

File: README.md (hunk 3 above)
Comment (afelisatti) [reply README.md]: Just to set some expectations that we might do it or not.

**Review by afelisatti — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jesica-fera) [reply README.md]: I changed this section a bit incorporating the comments from the editor on the marketplace doc

File: README.md [hunk 4]
```diff
@@ -1,28 +1,14 @@
...
-
-3. Release a version
-
-In order to release just create a tag with v<next_version> example if we want to release 0.0.4
-we just need to create v0.0.4 and push the tag. The CI should take care of the rest
-
-
-4. How to copy 
\ No newline at end of file
+MuleSoft welcomes you to submit issues, questions, comments, suggestions, and requests in this repositor
```
Comment (afelisatti) [inline README.md:14]:
```suggestion
MuleSoft welcomes you to submit issues, questions, comments, suggestions, and requests in this repository.
```

**Review by afelisatti — APPROVED** [review]

**Pull request merged**